features/FEATURE_BLE/ble/Gap.h

```diff
@@ -1062,7 +1068,7 @@ class Gap {
      * the address in input was not identifiable as a random address.
      */
     static ble_error_t getRandomAddressType(
-        BLEProtocol::AddressBytes_t address,
+        const BLEProtocol::AddressBytes_t address,
```

Comment:
ah, the evils of hiding pointers in typedefs :)

Reply:
The evil of array decaying to pointer and both being use interchangeably. Unfortunately references are not used in other signature which can lead incompatibility if the address passed in is a pointer and not an array.